### PR TITLE
Print column names in "stdout" output plugin

### PR DESCRIPTION
**Pull request by dmikurube**
No any "important" feature, but I wanted this when checking a column-renaming feature...

Can you have a look? @sakama

### CODE REVIEW

File: embulk-standards/src/main/java/org/embulk/standards/StdoutOutputPlugin.java [hunk 1]
```diff
@@ -40,6 +45,15 @@ public ConfigDiff transaction(ConfigSource config,
...
             OutputPlugin.Control control) {
         final PluginTask task = config.loadConfig(PluginTask.class);
+        if (task.getPrintsColumnNames()) {
+            for (final Column column : schema.getColumns()) {
+                if (column.getIndex() > 0) {
+                    System.out.print(",");
+                }
+                System.out.print(column.getName());
+            }
+            System.out.println("");
```
Comment (sakama) [inline embulk-standards/src/main/java/org/embulk/standards/StdoutOutputPlugin.java]: Why does this empty output is needed?
Is it intended to ":" ?

Comment (dmikurube) [reply embulk-standards/src/main/java/org/embulk/standards/StdoutOutputPlugin.java]: Ah, no. It's newline. One option was to use `\n`, but `println` might be better.

Comment (sakama) [reply embulk-standards/src/main/java/org/embulk/standards/StdoutOutputPlugin.java]: Ah Ok. I overlooked print"ln".

**Comment by dmikurube** [comment]
Thanks!

**Pull request merged**